engine/lib/relationships.php

```diff
@@ -7,22 +7,6 @@
  */
 
 /**
- * Convert a database row to a new \ElggRelationship
```

Comment:
Keeping a deprecated copy of this might be wise (and pose no overhead). I know it's private but I feel devs might be using `row_to_*` functions as callbacks, or within custom callbacks.